src/core/server/config/config_service.mock.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { BehaviorSubject } from 'rxjs';
+import { ObjectToConfigAdapter } from './object_to_config_adapter';
+
+import { ConfigService } from './config_service';
+
+type MethodKeysOf<T> = {
```

Comment:
Is there a separate place for declaring our custom TS helpers?

Reply:
Not yet. There are notions like "test utilities" elsewhere in the project that we could adopt in some form.

Do you anticipate using this helper in a lot of places?

Reply:
right now I inline it in every `*.mock.ts` file

Reply:
Ah, that's not great. We should come up with something, but it doesn't need to be in this PR.